package.json

```diff
@@ -27,12 +27,12 @@
     "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
     "mocha": "^9.0.1",
-    "standard": "^16.0.1",
-    "@prismarinejs/xboxlive-auth": "file:."
+    "prismarine-auth": "file:.",
+    "standard": "^16.0.1"
   },
   "dependencies": {
     "@azure/msal-node": "^1.1.0",
-    "@xboxreplay/xboxlive-auth": "^3.3.3",
+    "@xboxreplay/xboxlive-auth": "^4.0.0",
```

Comment:
I'm pretty sure this changes things internally, are you sure this works (tested in nmp/bedrock-protocol) ? No need to update if it's working fine as is.

Reply:
One of the main breaking changes is that authenticate method returns user_hash/xsts_token rather than userHash/XSTSToken.